README.md

```diff
@@ -74,6 +74,7 @@ See a [working example in codepen](https://codepen.io/potench/pen/GRZZprw).
 			publisherCountryCode: 'US',
 			// language: '', // empty string or unset to default to browser language
 			shouldUseStacks: true,
+			slimMode: true,
```

Comment:
i would stick with `should, can, has, is` prefix on booleans to stay consistent. 
```
isSlimMode
isFullWidth
```

Reply:
Updated boolean vars / function naming